### PR TITLE
Code coverage 100%

### LINKED ISSUE #44: Add test for datetime_must_have_timezone to get 100% Coverage
Coveralls says that we "only" have 99% coverage, because in the function `datetime_must_have_timezone` the branch that raises the `ValueError` isn't covered.

For nothing but fun, it would be nice to write a test case to cover that branch as well and thus reach 100% coverage. 😊 

https://github.com/openclimatefix/nowcasting_api/blob/f4ee7357bdc7be7ec862ce593b67c4e31f0b776a/src/main.py#L35-L39

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

test for last line not covered

## How Has This Been Tested?

unittests

- [ ] No
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

Fixes #44

### CODE REVIEW

File: src/tests/test_utils.py [hunk 1]
```diff
@@ -26,3 +30,43 @@ def get_every_minute():
         list_of_times.append(time_minutes)
         minutes += 1
     return list_of_times
+
+
+def test_datetime_must_have_timezone():
```
Comment (peterdudfield) [inline src/tests/test_utils.py:35]: This is the only new code

**Comment by flowirtz** [comment]
Haha wow I just raised an issue for this (#44) - glad you beat me to it! Thanks!

File: src/tests/test_utils.py [hunk 2]
```diff
@@ -26,3 +30,43 @@ def get_every_minute():
...
+    """Test function datetime_must_have_timezone"""
+
+    time_now = datetime.now(timezone.utc)
+
+    # check functions works
+    datetime_must_have_timezone(None, time_now)
+
+    with pytest.raises(Exception):
```
Comment (flowirtz) [inline src/tests/test_utils.py:43]: can we make this more specific and check it raises a `ValueError` rather than a generic Exception?

**Review by flowirtz — APPROVED** [review]
one small comment apart from that LGTM! Thanks!

**Pull request merged**